.github/workflows/codeql-analysis.yml

```diff
@@ -1,63 +0,0 @@
-name: 'CodeQL: Code Scanning'
```

Comment:
We should keep it, it runs on schedule and doesn't disrupt anything

Reply:
Recreated, although I am not aware of us ever getting any benefit from it?

Reply:
It output results here https://github.com/prisma/prisma/security/code-scanning?query=is%3Aopen+branch%3Amain+tool%3ACodeQL

So far nothing that needs to be taken care of, but one day it might find something important